domain/consensus/processes/blockvalidator/pruning_violation_proof_of_work_and_difficulty_test.go

```diff
@@ -101,7 +101,17 @@ func TestPOW(t *testing.T) {
 			t.Fatal(err)
 		}
 		random := rand.New(rand.NewSource(0))
-		mining.SolveBlock(validBlock, random)
+		// Difficulty is too high on mainnet to actually mine.
```

Comment:
What do we test if we skip pow? isn''t it better to skip the validBlock part for mainnet?

Reply:
you're probably right

Reply:
Done